app/src/main/java/org/mozilla/tv/firefox/navigationoverlay/BrowserNavigationOverlay.kt

```diff
@@ -134,6 +136,9 @@ class BrowserNavigationOverlay @JvmOverloads constructor(
 
     private lateinit var tileAdapter: PinnedTileAdapter
 
+    lateinit var pinnedTileViewModel: PinnedTileViewModel
+    lateinit var lifeCycleOwner: LifecycleOwner
```

Comment:
I just noticed that we don't use this anywhere in this class except to start observing it.  Suggest making `PinnedTileViewModel` a parameter of `#initTiles` and not maintaining a reference outside of that function.  

FWIW I'm doing this too and will be making the same change on the Pocket refactor.

Reply:
Like it